pages/advanced-algorithms/available-algorithms.md

```diff
@@ -30,6 +30,7 @@ library](/advanced-algorithms/install-mage).
 | [bipartite_matching](/advanced-algorithms/available-algorithms/bipartite_matching)                   | C++    | Algorithm for calculating maximum bipartite matching, where matching is a set of nodes chosen in such a way that no two edges share an endpoint.                                                                                  |
 | [bridges](/advanced-algorithms/available-algorithms/bridges)                                         | C++    | A bridge is an edge, which when deleted, increases the number of connected components. The goal of this algorithm is to detect edges that are bridges in a graph.                                                                 |
 | [community_detection](/advanced-algorithms/available-algorithms/community_detection)                 | C++    | The Louvain method for community detection is a greedy method for finding communities with maximum modularity in a graph. Runs in _O_(*n*log*n*) time.                                                                            |
+| [leiden_community_detection](/advanced-algorithms/available-algorithms/leiden_community_detection)          | C++    | The Leiden method for community detection is an improvement on the Louvain method, designed to find communities with maximum modularity in a graph while addressing issues of disconnected communities. Runs in _O_(*L* *m*)  time, where *L* is the number of iterations of the algorithm
```

Comment:
Not in alphabetical order

Reply:
Not sure why you resolved this one. 

Perhaps one solution would be to rename from `leiden_community_detection` to `community_detection_leiden` so the two sort adjacent to each other.